azure-functions/acbs-function/helpers/mandatoryFields.js

```diff
@@ -1,5 +1,12 @@
-const findMissingMandatory = (data, mandatoryFields) =>
-  mandatoryFields.map((mf) => (data[mf] !== null || data[mf] !== undefined || data[mf] !== '' ? '' : mf)).filter((mf) => mf !== '');
+/**
+ * Finds missing mandatory fields in the provided data object.
+ *
+ * @param {Object} data - The data object to check for missing fields.
```

Comment:
```suggestion
 * @param {object} data - The data object to check for missing fields.
```

Reply:
We have a mix/match with Array/array as well, take your pick for this PR I guess 😄 

```ts
@param {Array} - 46 instances
@param {array} - 14 instances
```

Reply:
Should be `Array` I believe

Reply:
Have updated all and merged to `main` under PR #3600